docs/release_notes/release_notes-0.15.0.rst

```diff
@@ -22,7 +24,7 @@ Key Features and Updates
 * Update testing suite
   *
 * Documentation improvements
-  *
+  * DOCS-#4296: Fix docs warnings (#4297)
```

Comment:
Please add yourself in Contributors list below.

Reply:
`amyskov` was added to the Contributors list.